README.md

```diff
@@ -3,35 +3,82 @@
 ## Overview
 The edX learning platform's frontend for enterprise learners.
 
-## Getting started
+## Setup for development
 
-- Clone this repository locally with `git clone <repo URL>`
```

Comment:
With the npm stuff still being in the file (and some additions) right after this line, I don't believe this should be deleted.

This specific repo is not downloaded by devstack atm, so you still need to clone it before you would be able to run npm start, unless npm start is referring to something else here (which if it is, is not clear)